catalyst_voices/lib/widgets/separators/voices_named_divider.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:flutter/material.dart';
+
+/// Voices variation of [Divider] that requires name in middle of divider.
+class VoicesNamedDivider extends StatelessWidget {
```

Comment:
Did designers called it NamedDivider? I most often seen this kind of widget as 'TextDivider' etc

Reply:
Hmmm interesting, `NamedDivider` sound more natural to me tho but works both ways. Ill rename it